### PR TITLE
fixed #457

### LINKED ISSUE #457: Remove the unwanted 20px padding below the menu bar on Windows
Remove the space shown in the below screenshot:
Screenshot 2020-09-17 at 8 01 51 AM

### PR DESCRIPTION
**Pull request by MBakirci**

### CODE REVIEW

File: desktop-app/app/components/Header/index.js [hunk 1]
```diff
@@ -14,7 +14,6 @@ import Logo from '../icons/Logo';
 
 const Header = () => {
   const classes = useStyles();
-  const theme = useTheme();
```
Comment (jjavierdguezas) [inline desktop-app/app/components/Header/index.js:17]: if you removed this, please remove the `import` of `useTheme` on line `4`

**Review by jjavierdguezas — CHANGES_REQUESTED** [review]
Looks good @MBakirci thanks
please comment something in the issue to assign it to you

@MBakirci you are only handling this for Windows but I saw that space on Ubuntu too, should we address that too? @manojVivek 
5bd1bd1a-fc43-4687-ae9f-e1d010728cc3

**Comment by MBakirci** [comment]
Thanks @jjavierdguezas. That is a good one, I only tested Responsively on Windows and Mac, but Linux has a lot of distros. I changed the line to `os.platform() === 'darwin' ?  '20px 0 5px' : '0 0 0'` and it works great on Ubuntu, but I don't know how it works on other distros.

**Comment by jjavierdguezas** [comment]
@MBakirci I think this is OK, please comment in the issue #457 to assign the issue to you

**Review by jjavierdguezas — APPROVED** [review]

**Comment by jjavierdguezas** [comment]
@MBakirci Thanks so much for your code contribution.
Looking forward to more collaboration in the future! 🙌

**Comment by jjavierdguezas** [comment]
@all-contributors Please add @MBakirci for code.

**Comment by jjavierdguezas** [comment]
@MBakirci Do you might sharing your Twitter handle? We would love to send a shout out for your work on twitter. :)

**Comment by manojVivek** [comment]
@MBakirci Thanks for the contribution! 

Please join our slack workspace for any future discussions: https://join.slack.com/t/responsively/shared_invite/zt-haoieftz-IsMw64H6jXC23pJ16ROLzw :)

**Pull request merged**